sechub-administration/src/main/java/com/daimler/sechub/domain/administration/project/ProjectAdministrationRestController.java

```diff
@@ -95,53 +99,62 @@ public void createProject(@RequestBody @Valid ProjectJsonInput input) {
 		}
 
 		/* @formatter:on */
-		creationService.createProject(input.getName(), input.getDescription(), input.getOwner(), whiteListedURIs, metaData);
-	}
+        creationService.createProject(input.getName(), input.getDescription(), input.getOwner(), whiteListedURIs, metaData);
+    }
 
-	/* @formatter:off */
+    /* @formatter:off */
 	@UseCaseAdministratorShowsProjectDetails(@Step(number = 1,name="Rest call", description = "Json returned containing details about project", needsRestDoc = true))
 	@RequestMapping(path = AdministrationAPIConstants.API_SHOW_PROJECT_DETAILS, method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
 	public ProjectDetailInformation showProjectDetails(@PathVariable(name="projectId") String projectId) {
 		/* @formatter:on */
-		return detailsService.fetchDetails(projectId);
-	}
+        return detailsService.fetchDetails(projectId);
+    }
 
-	/* @formatter:off */
+    /* @formatter:off */
 	@UseCaseAdministratorListsAllProjects(@Step(number=1,name="Rest call",description="All project ids of sechub are returned as json", needsRestDoc=true))
 	@RequestMapping(path = AdministrationAPIConstants.API_LIST_ALL_PROJECTS, method = RequestMethod.GET, produces= {MediaType.APPLICATION_JSON_VALUE})
 	public List<String> listProjects() {
 		/* @formatter:on */
-		return repository.findAll().stream().map(Project::getId).collect(Collectors.toList());
-	}
-
-	/* @formatter:off */
+        return repository.findAll().stream().map(Project::getId).collect(Collectors.toList());
+    }
+
+    /* @formatter:off */
+	@UseCaseAdministratorAssignsOwnerToProject(@Step(number=1,name="Rest call",description="Administrator does call rest API to assign new owner",needsRestDoc=true))
+    @RequestMapping(path = AdministrationAPIConstants.API_ASSIGN_OWNER_TO_PROJECT, method = RequestMethod.POST, produces= {MediaType.APPLICATION_JSON_VALUE})
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
I chose CREATED because of consistency to `assignUserToProject`. From REST-API user perspective every POST-request should create or change something on server side. The user has no deeper insight into whether some internal resource is created or changed with assignOwner as well as with assignUser.

In my opinion, both requests should have the same ResponseStatus. OK would be also suitable, because in both cases no user facing resource is created but already existing resources are altered.

Should I change both to HttpStatus.OK?

Reply:
Issue #567 created. Will be implemented in this PR.

Reply:
OK.